### DISCUSSION TITLE
SMART Configuration endpoint not available?

### DISCUSSION THREAD

**Discussion by bruceharrison1984** [FirelyTeam/firely-net-sdk#2062] [Q&A] [answered]
I'm trying to get SMART working for preliminary Inferno tests, but I cannot access the `http://localhost:4080/.well-known/smart-configuration` endpoint. It simply returns a FHIR 404 error.

I've got the `Vonk.Smart` plugin added to my config file, and I've added `SmartAuthorizationOptions` to appsettings.instance.json as well. Am I missing some other step so I can access this endpoint? 

The documentation makes it seem like this is all I should need for basic operation, though the docs are a little thin around this aspect. The only reference I could find to this endpoint was in previous release notes: https://docs.fire.ly/projects/Firely-Server/en/latest/releasenotes/releasenotes.html#id43

**Comment by alexzautke** [comment, marked as answer]
Hi @bruceharrison1984,

thanks for reaching out to us. In order to help you set up Firely Server with SMART on FHIR enabled, we would need a copy of your appsettings. Could you send the appsettings.instance.json and the start-up log file to server@fire.ly?

The .well-known/smart-configuration should be available instantly without needing to change further settings. Do you see any error messages related to the authorization server in the log file? The smart-configuration won't be available if the authorization server cannot be reached.

Thanks for the remark about our documentation. I'll make sure that the documentation regarding the smart-configuration will be updated.

For future issues, please feel free to reach out directly to server@fire.ly, as this discussion board is only meant for issues for the Firely .NET SDK.

**Reply by bruceharrison1984** [reply]
Thank you for the quick reply. I'll get everything sent over tomorrow. 👍

**Reply by bruceharrison1984** [reply]
I ended up getting things working. My issue was placing the SMART configuration into my `appsetttings.instance.json` within my project instead of `appsettings.default.json` under C:/FirelyServer. Once I moved the configuration settings, things started working as expected. I definitely think the docs could use a bit more detail around this aspect, but great job on this none-the-less!
